src/v/raft/heartbeat_manager.cc

```diff
@@ -101,8 +101,7 @@ static heartbeat_requests requests_for_range(
             }
 
             auto seq_id = ptr->next_follower_sequence(rni);
-            ptr->update_suppress_heartbeats(
-              rni, seq_id, heartbeats_suppressed::yes);
+
```

Comment:
this does not seem to belong in this patch that swaps out flat_hash_map for node_hash_map